src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/GerritServer/index.jelly

```diff
@@ -230,6 +230,31 @@
                                         checked="${it.config.enableManualTrigger}"
                                         default="${true}"/>
                         </f:entry>
+                        <f:entry title="${%Fetch Project List Settings}"
+                                 help="/plugin/gerrit-trigger/help-GerritFetchProjectList.html">
```

Comment:
optionalBlock works great when the checkbox should control the visibility of other elements. But in this case the textbox is _not_ dependent on the checkbox. E.g. I would like both to be visible. So I used the <divs> since I really wanted to align the elements in rows and using a table would not work at all (f:entry + table = broken form submission. I tried everything. It seems to work in hetrolists and repeatables, but not directly under  <f:entries>).

So I'm out of ideas. I see tables are used extensively by the Gerrit-Trigger. While it is not optimal, I'm guessing we can make exceptions when the core libs do not want to "play ball"?